charts/rancher-monitoring/105.0.0-rc1+up57.0.3/charts/prometheus-adapter/templates/deployment.yaml

```diff
@@ -60,7 +60,6 @@ spec:
         - --tls-private-key-file=/var/run/serving-cert/tls.key
         {{- end }}
         - --cert-dir=/tmp/cert
-        - --logtostderr=true
```

Comment:
Does this change the behavior at all? or are logs sent to stderr be default?

Reply:
According to [KEP-2845](https://github.com/kubernetes/enhancements/blob/master/keps/sig-instrumentation/2845-deprecate-klog-specific-flags-in-k8s-components/README.md#logging-defaults) the default is to split logs into two streams, stdout for info logs and stderr for error logs.